packages/accounts/src/light-account/client.ts

```diff
@@ -19,81 +18,49 @@ import {
 } from "./decorator.js";
 
 export type CreateLightAccountClientParams<
-  TEntryPointVersion extends EntryPointVersion,
+  TTransport extends Transport = Transport,
```

Comment:
don't think this change is correct.

Reply:
this is what it was before https://github.com/alchemyplatform/aa-sdk/blob/main/packages/accounts/src/light-account/client.ts#L21